android/src/test/resources/local/testAppSpec.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Why are you committing this file when it already exists in this repo https://github.com/ably/ably-java/blob/master/lib/src/test/resources/local/testAppSpec.json?  From what I can tell, your build can simply copy when running the build instead?  I don't see any good reason to have two copies of the same file in the repo.